doc/source/user-experience.rst

```diff
@@ -253,6 +253,16 @@ the guarantee to allow some users to use larger amounts of RAM for
 a very short-term time (e.g. when running a single, short-lived function that
 consumes a lot of memory).
 
+Similarly, you can limit CPU as follows:
+    .. code-block:: yaml
+
+       singleuser:
+           cpu:
+              limit: 500m
+              guarantee: 500m
+              
+This would limit your users to a maximum of 500 "milli-cpu" (so 1/2 of a CPU core), as well as guarantee them that same amount - this measure is defined in the kubernetes docs here: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu .
```

Comment:
I think as a general rule we're hyperlinking text rather than displaying full links. Could you do something like "this measure is defined in the compute resources page of the Kubernetes docs" and hyperlink "compute resources page"?